inline-c-cpp/inline-c-cpp.cabal

```diff
@@ -9,9 +10,8 @@ author:              Francesco Mazzoli
 maintainer:          f@mazzo.li
 copyright:           (c) 2015-2016 FP Complete Corporation, (c) 2017-2019 Francesco Mazzoli
 category:            FFI
-tested-with:         GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.5, GHC == 8.8.4, GHC == 8.10.2
+tested-with:         GHC == 8.4.4, GHC == 8.6.5, GHC == 8.8.4, GHC == 8.10.2
```

Comment:
Please write in the relevant commit message what feature made us drop GHC 8.2.

Reply:
Done

> LTS 11 comes with Cabal < 2.2, which doesn't support cxx-options.